index.html

```diff
@@ -30,14 +30,186 @@ <h1 class="info_box">
     </div>
     <div>
         <p class="contact_section"><a href="#">LET'S CONNECT</a></p>
+        
     </div>
+    <section>
         <ul>
-            <li><a href="#"><img src="cat.svg" width="1" height="15" alt="cat"></a></li>
-            <li><a href="#"><img src="bird.svg" width="1" height="15" alt="bird"></a></li>
-            <li><a href="#"><img src="m.svg" width="1" height="15" alt="M"></a></li>
-            <li><a href="#"><img src="n.png" width="1" height="15" alt="N"></a></li>
-            <li><a href="#"><img src="two.svg" width="1" height="15" alt="two"></a></li>
-         </ul>
-    <h3><a href="#">LET'S GET INTERACTIVE</a></h3>
+            <li><a href="#"><img src="/img/cat.svg" width="1" height="15" alt="cat"></a></li>
+            <li><a href="#"><img src="/img/bird.svg" width="1" height="15" alt="bird"></a></li>
+            <li><a href="#"><img src="/img/m.svg" width="1" height="15" alt="M"></a></li>
+            <li><a href="#"><img src="/img/n.png" width="1" height="15" alt="N"></a></li>
+            <li><a href="#"><img src="/img/two.svg" width="1" height="15" alt="two"></a></li>
+        </ul>
+        <h3><a href="#">LET'S GET INTERACTIVE</a></h3>
+    </section>
+
+        
+
+    <section class="section_works_container">
+       <div>
+            <img class="sec_in" src="/img/sndwk.jpg" alt="selfie">
+            <div class="tonic">
+                <h2>Tonic</h2>
+            </div>
+            <div>
+                <ul class="sec_h">
+                    <li><a href="#"><img src="/img/canopy.svg" alt="canopy"></a></li>
+                    <li><a href="#"><img src="/img/Count.svg" width="1" height="5" alt="count"></a></li>
+                    <li><a href="#"><img src="/img/bed.svg" height="22" alt="bed"></a></li>
+                    <li><a href="#"><img src="/img/cc.svg" width="1" height="5" alt="dot"></a></li>
+                    <li><a href="#"><img src="/img/time.svg" alt="date"></a></li>
+                </ul>
+            </div>
+        </div>
+        <section>
+            <h3 class="p">
+                A daily selection of privately personalized reads; no accounts or sign-ups required.
+            </h3>
+            <div>
+                <ul class="push">
+                    <li class="html"><a href="#">html</a></li>
+                    <li class="css"><a href="#">CSS</a></li>
+                    <li class="javasc"><a href="#">Javascript</a></li>
+                </ul>
```

Comment:
As said the previous reviewer, please stick to Figma:
On figma to find what styles to use, just click on the element, and on the right you will see the CSS. property (size, color ...) 

your
<img width="241" alt="image" src="https://user-images.githubusercontent.com/7243132/170830384-2d2c8cfa-76c9-4376-8d62-cf505ff42335.png">

Figma:
<img width="241" alt="image" src="https://user-images.githubusercontent.com/7243132/170830397-4c33ba49-f9aa-4b05-a231-d56338f98800.png">